Cybereason malop processes bug fix (#12127)
demisto/content: Packs/Cybereason/ReleaseNotes/1_0_7.md

#### Integrations
##### Cybereason

> - Fixed an issue where the command ***cybereason-malop-processes*** used to work in previous version but stopped working.

Reviewer: ```suggestion
- Fixed an issue where the ***cybereason-malop-processes*** command was not showing the proper outputs.
```
